<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ctkXnatLoginDialog</class>
 <widget class="QDialog" name="ctkXnatLoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login Profiles</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,3">
   <item>
    <widget class="QWidget" name="wgtProfiles" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="lblProfiles">
        <property name="text">
         <string>Profiles:</string>
        </property>
        <property name="buddy">
         <cstring>lstProfiles</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListView" name="lstProfiles">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="btnSave">
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDelete">
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="wgtFields" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Login information:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lblProfileName">
           <property name="text">
            <string>Profile name:</string>
           </property>
           <property name="buddy">
            <cstring>edtProfileName</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="edtProfileName"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lblServerUri">
           <property name="text">
            <string>Server URI:</string>
           </property>
           <property name="buddy">
            <cstring>edtServerUri</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="edtServerUri"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="lblUserName">
           <property name="text">
            <string>User name:</string>
           </property>
           <property name="buddy">
            <cstring>edtUserName</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="edtUserName"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="lblPassword">
           <property name="text">
            <string>Password:</string>
           </property>
           <property name="buddy">
            <cstring>edtPassword</cstring>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="edtPassword">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QCheckBox" name="cbxDefaultProfile">
           <property name="text">
            <string>Use as default</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="bbxDialog">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edtProfileName</tabstop>
  <tabstop>edtServerUri</tabstop>
  <tabstop>edtUserName</tabstop>
  <tabstop>edtPassword</tabstop>
  <tabstop>cbxDefaultProfile</tabstop>
  <tabstop>btnSave</tabstop>
  <tabstop>btnDelete</tabstop>
  <tabstop>lstProfiles</tabstop>
  <tabstop>bbxDialog</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>bbxDialog</sender>
   <signal>accepted()</signal>
   <receiver>ctkXnatLoginDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bbxDialog</sender>
   <signal>rejected()</signal>
   <receiver>ctkXnatLoginDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
